free-programming-books-es.md

```diff
@@ -164,7 +164,9 @@
 
 ### Go
 
+* [Aprende-GO](https://github.com/JJ/aprende-go)
```

Comment:
This looks like an incomplete book that was abandoned. We could list it with an `under construction` note if there's a chance it will be picked up.
Please add author names where appropriate.